### PR TITLE
Update demo files

### PR DESCRIPTION
**Pull request by alexshoe**
- Fixes "unexpected argument" errors in `usage.py` and `usage-advanced.py`
- Replaced deprecated import calls to match new Dash 2.0 standards
- Replaced broken video links with working URLs

I have rebuilt and run `usage.py`, `usage-method.py`, and `usage-advanced.py` - everything works as expected.

### CODE REVIEW

File: usage-advanced.py [hunk 1]
```diff
@@ -2,226 +2,215 @@
...
 import dash
-import dash_html_components as html
-import dash_core_components as dcc
-from dash.dependencies import Input, Output, State
+from dash import dcc, html, Input, Output, State
 
 app = dash.Dash(__name__)
 server = app.server
 
 app.scripts.config.serve_locally = True
```
Comment (alexcjohnson) [inline usage-advanced.py:10]: 🔪 this is the default now (and can be a `Dash` constructor arg anyway if you want to change it)

File: usage-advanced.py [hunk 2]
```diff
@@ -2,226 +2,215 @@
...
+    ]
+)
+
+
+@app.callback(
+    Output("video-player", "url"),
+    [Input("button-update-url", "n_clicks")],
+    [State("input-url", "value")],
+    prevent_initial_call=True,
+)
```
Comment (alexcjohnson) [inline usage-advanced.py:132]:
```suggestion
@callback(
    Output("video-player", "url"),
    Input("button-update-url", "n_clicks"),
    State("input-url", "value"),
    prevent_initial_call=True,
)
```
Doesn't have to be in this PR, but at some point the rest of this syntax should be updated as well :)

File: usage-advanced.py [hunk 3]
```diff
@@ -2,226 +2,215 @@
...
+@app.callback(Output("video-player", "seekTo"), [Input("slider-seek-to", "value")])
 def set_seekTo(value):
     return value
 
 
-if __name__ == '__main__':
-    app.run_server(debug=True)
\ No newline at end of file
+if __name__ == "__main__":
+    app.run_server(debug=True)
```
Comment (alexcjohnson) [inline usage-advanced.py:216]:
```suggestion
    app.run(debug=True)
```

File: usage-advanced.py [hunk 4]
```diff
@@ -2,226 +2,215 @@
 
 import dash_player
 import dash
-import dash_html_components as html
-import dash_core_components as dcc
-from dash.dependencies import Input, Output, State
+from dash import dcc, html, Input, Output, State
```
Comment (alexcjohnson) [inline usage-advanced.py:5]: Generally discouraged to import the same package two ways - include `Dash` in the named imports (and `callback` per my other comment) and the `import dash` can be removed.

**Review by alexcjohnson — APPROVED** [review]
💃 Looks good, up to you if you want to tackle the other stylistic udpates in this PR or another.

File: usage-advanced.py (hunk 4 above)
Comment (alexshoe) [reply usage-advanced.py:5]: @alexcjohnson I never knew that, thanks for the heads up!

**Pull request merged**